loader/loader_test.go

```diff
@@ -879,6 +889,15 @@ func TestSplitPrefix(t *testing.T) {
 			wantParts: []string{"x", "y"},
 			wantPath:  "file:///c:/a/b/c",
 		},
+		{
+			input:    "c:/a/b/c",
+			wantPath: "c:/a/b/c",
```

Comment:
This doesn't seem compatible with how data prefixes work in OPA, as we'd expect `c` to become the data prefix.